controllers/admin/AdminImportController.php

```diff
@@ -118,47 +118,106 @@ public function __construct()
                 );
 
                 $this->available_fields = array(
-                    'no' => array('label' => $this->trans('Ignore this column', array(), 'Admin.Advparameters.Feature')),
-                    'id_product' => array('label' => $this->trans('Product ID', array(), 'Admin.Advparameters.Feature')),
-                    'product_reference' => array('label' => $this->trans('Product Reference', array(), 'Admin.Advparameters.Feature')),
+                    'no' => array(
+                        'label' => $this->trans('Ignore this column', array(), 'Admin.Advparameters.Feature'),
+                    ),
+                    'id_product' => array(
+                        'label' => $this->trans('Product ID', array(), 'Admin.Advparameters.Feature'),
+                    ),
+                    'product_reference' => array(
+                        'label' => $this->trans('Product Reference', array(), 'Admin.Advparameters.Feature'),
+                    ),
                     'group' => array(
-                        'label' => $this->trans('Attribute (Name:Type:Position)', array(), 'Admin.Advparameters.Feature').'*'
+                        'label' => $this->trans(
+                            'Attribute (Name:Type:Position)',
+                            array(),
+                            'Admin.Advparameters.Feature'
+                        ) . '*',
```

Comment:
- Around 200 occurrences of "` . `"in /src/ (new code)
- Around 7500 occurrences in the whole project (including generated php code in cache)
- Regular use of spaces by @julienbourdeau, @rGaillard, @thierrymarianne, @antoin-m, @fatmaBouchekoua (should I name them all ?) and community contributors (meaning that you guys validated it).
- Internet debated about if for long. Even at Symfony they tend not to strictly stick to this rule. Other big framework and communities are using spaces around concatenation operator.
- When I tried to talk about it with core devs, I had no clear answer. It seems like everybody tries his best to not break what he sees in the file, file by file, case by case. It also seems like any single people I asked prefers using spaces (including me).

So yes, all this is unclear.